app/uk/gov/hmrc/helptosavestub/controllers/EligibilityCheckController.scala

```diff
@@ -42,32 +43,55 @@ class EligibilityCheckController extends BaseController {
     8 → "Entitled to WTC and in receipt of positive WTC/CTC Tax Credit and in receipt of UC and income sufficient"
   )
 
-  val eligibleReasonGen: Gen[EligibilityCheckResult] =
-    Gen.choose(6, 8) // scalastyle:ignore magic.number
-      .map{ i ⇒
-        val reason = reasonMappings.getOrElse(i, sys.error(s"Could not find eligibility reason for code $i"))
-        EligibilityCheckResult("Eligible to HtS Account", 1, reason, i)
-      }
-
-  val ineligibleReasonGen: Gen[EligibilityCheckResult] =
-    Gen.choose(2, 5) // scalastyle:ignore magic.number
-      .map{ i ⇒
-        val reason = reasonMappings.getOrElse(i, sys.error(s"Could not find ineligibility reason for code $i"))
-        EligibilityCheckResult("Ineligible to HtS Account", 2, reason, i)
-      }
-
   val alreadyHasAccountResult: EligibilityCheckResult =
     EligibilityCheckResult("HtS account already exists", 3, "HtS account was previously created", 1)
 
+  val invalidResultCode: EligibilityCheckResult =
```

Comment:
no unit tests for all these changes ?

Reply:
Can I seek your help with some unit tests, please?

Reply:
sure, i will help you on monday ?

Reply:
Sounds good - thanks Suresh.